### PR TITLE
resolving the conflict that xts package makes in data.atmosphere

### PR DESCRIPTION
**Pull request by para2x**
In this PR I just took the `xts` package out of the Description file in the data.atmosphere module because it was masking some of the functions of `dplyr`.

## Review Time Estimate

- [ ] Immediately
- [ ] Within one week
- [ ] When possible
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue) 
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the CHANGELOG.md.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: modules/data.atmosphere/DESCRIPTION [hunk 1]
```diff
@@ -16,8 +16,7 @@ Description: The Predictive Ecosystem Carbon Analyzer (PEcAn) is a scientific
     integrated into PEcAn. As a standalone package, it provides an interface to
     access diverse climate data sets.
 Depends:
-    methods,
-    xts
+    methods
```
Comment (infotroph) [inline modules/data.atmosphere/DESCRIPTION]: 👍

Comment (para2x) [reply modules/data.atmosphere/DESCRIPTION]: I'll make that change too.

**Comment by para2x** [comment]
@infotroph @mdietze I think this should be reay to be pulled in .

**Comment by infotroph** [comment]
Mike beat me to approving, but I also approve. Thanks!

**Pull request merged**